src/data/markdown/docs/02 javascript api/08 k6-http/10-batch- requests -.md

```diff
@@ -12,19 +12,20 @@ Batch multiple HTTP requests together, to issue them in parallel over multiple T
 
 When each request is specified as an array, the order of the arguments for each request is as follows:
 
-### Returns
-
-| Type   | Description                                                                                                                                                                                                                   |
-| ------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| object | An object containing [Response](/javascript-api/k6-http/response) objects.<br /><br />It is an array when users pass an array as `requests` and is a normal object with string keys when named requests are used (see below). |
-
 | Position | Name              | Type             | Description                                                                                                                 |
 | -------- | ----------------- | ---------------- | --------------------------------------------------------------------------------------------------------------------------- |
 | 1        | method            | string           | Mandatory. The HTTP method of the request. One of GET, POST, PUT, PATCH, DELETE, HEAD or OPTION.                            |
-| 2        | url               | string           | Mandatory. The URL to request.                                                                                              |
+| 2        | url               | string / [HTTP URL](/javascript-api/k6-http/url-url#returns)  | Mandatory. The URL to request.                                                                                              |
 | 3        | body (optional)   | string / object / ArrayBuffer | The body of the request if relevant. Can be set to `null` if not applicable but you want to set the last `params` argument. |
 | 4        | params (optional) | object           | [Params](/javascript-api/k6-http/params) like auth, custom headers and tags.                                                |
 
+
+### Returns
+
+| Type   | Description                                                                                                                                                                                                                   |
+| ------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| object | An object containing [Response](/javascript-api/k6-http/response) objects.<br /><br />It is an array when users pass an array as `requests` and is a normal object with string keys when named requests are used (see below). |
+
```

Comment:
Change unrelated to `http.url`, but seemed like a correct one?

Before:
<img width="769" alt="Screenshot 2021-10-06 at 09 47 42" src="https://user-images.githubusercontent.com/10167318/136153700-b41e455d-4549-4b49-ac64-847aa800edb1.png">

After:
<img width="761" alt="Screenshot 2021-10-06 at 09 53 23" src="https://user-images.githubusercontent.com/10167318/136154386-8e6ce7dc-842b-4aef-b7b8-0d272d36fb67.png">

Reply:
**A tiny suggestion:**

_The returned_ object _contains_ Response objects.

It is an array when users pass an array as `requests` and is _an ordinary object_ with string keys when named requests are used (see below).